tensorflow_lite_support/ios/task/vision/sources/TFLObjectDetector.m

```diff
@@ -88,6 +97,14 @@ + (nullable instancetype)objectDetectorWithOptions:(nonnull TFLObjectDetectorOpt
 
 - (nullable TFLDetectionResult *)detectWithGMLImage:(GMLImage *)image
                                               error:(NSError *_Nullable *)error {
+
+  if (!image) {
```

Comment:
@khanhlvg Thoughts on this as well. Based on my comment [here](https://github.com/tensorflow/tflite-support/pull/787/files#r849323481)  and [here](https://github.com/tensorflow/tflite-support/pull/787/files#r849324803).

Reply:
Yes, makes sense.